### ISSUE TITLE
Vagrant error: replacing SSH key

### ISSUE THREAD

**Issue by brookingcharlie** [Bahmni/bahmni-vagrant#8]
When running `vagrant up` on OSX, I get an error: "Timed out while waiting for the machine to boot." See full error listing at the end of this issue.

It turns out that Vagrant is unable to SSH to the box after replacing the insecure SSH key.

It can be fixed by modifying the `Vagrantfile` as follows. I'm not sure whether this is an acceptable fix for Bahmni/bahmni-vagrant or something else in the tooling can be upgraded.

```
diff --git a/Vagrantfile b/Vagrantfile
index 490329a..b7fc704 100644
--- a/Vagrantfile
+++ b/Vagrantfile
@@ -3,6 +3,7 @@

 Vagrant.configure(2) do |config|
   config.vm.box = "bahmni-team/bahmni"
+  config.ssh.insert_key = false
   config.vm.box_check_update = true
   config.vm.network "private_network", ip: "192.168.33.10"
```

Environment:
- bahmni-vagrant commit: a6872da
- Vagrant box: bahmni-team/bahmni  (virtualbox, 0.83)
- Installed: vagrant: 1.8.5; virtualbox: 5.1.4-110228
- OS: Darwin Kernel Version 15.6.0

Full error listing:

```
$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'bahmni-team/bahmni'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'bahmni-team/bahmni' is up to date...
==> default: Setting the name of the VM: bahmni-vagrant_default_1472036968010_84673
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostly
==> default: Forwarding ports...
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customisations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: 
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default: 
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
    default: Warning: Authentication failure. Retrying...
Timed out while waiting for the machine to boot. This means that
Vagrant was unable to communicate with the guest machine within
the configured ("config.vm.boot_timeout" value) time period.

If you look above, you should be able to see the error(s) that
Vagrant had when attempting to connect to the machine. These errors
are usually good hints as to what may be wrong.

If you're using a custom box, make sure that networking is properly
working and you're able to connect to the machine. It is a common
problem that networking isn't setup properly in these boxes.
Verify that authentication configurations are also setup properly,
as well.

If the box appears to be booting properly, you may want to increase
the timeout ("config.vm.boot_timeout") value.
```

**Comment by sravanthi17** [comment]
Thanks for pointing out. Updated the Vagrantfile in master and will be available from v0.84.

**Comment by javagovindsharma** [comment]
Thanks
